packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -66,61 +59,65 @@ const styles: ComponentStyles<ToggleClassNameContract, DesignSystem> = (
             width: "44px",
             height: "20px",
-            ...outlinePattern.rest,
+            background: neutralFillInputRest,
+            border: `${toPx(
+                designSystem.outlinePatternOutlineWidth
+            )} solid ${neutralOutlineRest(designSystem)}`,
             borderRadius: "20px",
             appearance: "none",
             outline: "none",
             "&:hover": {
-                ...outlinePattern.hover,
+                background: neutralFillInputHover,
+                border: `${toPx(
+                    designSystem.outlinePatternOutlineWidth
+                )} solid ${neutralOutlineHover(designSystem)}`,
+            },
+            "&:active": {
+                background: neutralFillInputActive,
+                border: `${toPx(
+                    designSystem.outlinePatternOutlineWidth
+                )} solid ${neutralOutlineActive(designSystem)}`,
             },
             ...applyFocusVisible({
-                ...outlinePattern.focus,
+                boxShadow: `0 0 0 1px ${designSystem.foregroundColor} inset`,
+                border: `${toPx(designSystem.outlinePatternOutlineWidth)} solid ${
+                    designSystem.foregroundColor
+                }`,
             }),
         },
-        toggle__disabled: {
-            ...typographyPattern.disabled,
-            "& $toggle_input, & $toggle_label, & $toggle_statusMessage": {
-                cursor: "not-allowed",
-            },
+        toggle__checked: {
             "& $toggle_input": {
-                ...outlinePattern.disabled,
-            },
-            "& $toggle_stateIndicator": {
-                ...switchFieldPattern.disabled.stateIndicator,
-                "@media screen and (-ms-high-contrast:active)": {
-                    background: "ButtonText",
-                },
-            },
-            "&$toggle__checked": {
-                "& $toggle_input": {
-                    ...outlinePattern.disabled,
-                    borderColor: "transparent",
-                    ...switchFieldPattern.disabled.stateIndicator,
-                },
-                "& $toggle_stateIndicator": {
-                    background: backgroundColor,
+                background: accentFillRest,
+                borderColor: accentFillRest,
+                "& + $toggle_stateIndicator": {
+                    left: "28px",
+                    background: accentForegroundCut,
                 },
             },
         },
-        toggle__checked: {
+        toggle__disabled: {
+            cursor: "not-allowed",
+            opacity: "0.3",
             "& $toggle_input": {
-                backgroundColor: brandColor,
-                borderColor: brandColor,
-                ...applyFocusVisible({
-                    ...outlinePattern.focus,
-                }),
+                background: neutralFillSelected,
+                borderColor: neutralFillSelected,
                 "& + $toggle_stateIndicator": {

```

Comment:
Same question for all of these - I don't *think* you need to select as a sibling of `$toggle_input` - I think you can just do normal child selectors.